libafl/src/executors/inprocess/mod.rs

```diff
@@ -63,13 +80,22 @@ where
     OT: ObserversTuple<S>,
     S: State,
 {
-    /// The harness function, being executed for each fuzzing loop execution
     harness_fn: HB,
-    /// The observers, observing each run
-    observers: OT,
-    // Crash and timeout hah
-    hooks: (InProcessHooks, HT),
-    phantom: PhantomData<(S, *const H)>,
+    inner: GenericInProcessExecutorInner<HT, OT, S>,
```

Comment:
what was the reason that you have to make this inner?
i think i heard once but forgot

Reply:
It's to avoid code duplication between GenericInProcessExecutor and GenericInProcessExecutorWithState.

Reply:
ok